src/mudlet.cpp

```diff
@@ -121,7 +121,6 @@ mudlet* mudlet::self()
     return _self;
 }
 
-
```

Comment:
If you put that blank line back then that is one less file touched...

Reply:
It seems we have an extra line in that file, not a big deal to remove it I think.